### PR TITLE
Issue 10416 - input-group buttons not sized properly

### PR DESCRIPTION
**Pull request by gskaplan**
This fix resolve the issue where input-group-buttons aren't filling up the entire space of the parent container. Thanks to @rafibomb for helping with the code. This relates to issue #10416.

Please review as necessary. @IamManchanda.

### CODE REVIEW

**Comment by IamManchanda** [comment]
Thanks digging it in!

**Comment by IamManchanda** [comment]
@gskaplan Please rename `input-group-flow.html` to `input-group-float.html`

**Comment by gskaplan** [comment]
Renamed and new PR created.

File: scss/forms/_input-group.scss [hunk 1]
```diff
@@ -107,6 +107,7 @@ $input-prefix-padding: 1rem !default;
 
     @if $global-flexbox {
       flex: 0 0 auto;
+      display: flex;
```
Comment (IamManchanda) [inline scss/forms/_input-group.scss]: Display first and then `flex` shorthand property 
See https://github.com/zurb/foundation-sites/blob/develop/.sass-lint.yml#L151-L153

**Comment by IamManchanda** [comment]
Thanks... 
cc @kball

**Comment by kball** [comment]
Excellent work. Thanks @gskaplan! And congratulations on first merged PR!

**Pull request merged**